BUILD

```diff
@@ -292,3 +314,19 @@ upgrading GCC. If you have libstdc++ corresponding to GCC 5.x or 6.x, you *must*
 old ABI, but Dinit will be broken if you upgrade to GCC 7. If you have libstdc++ from GCC 7, you
 *must* build with the new ABI. If the wrong ABI is used, Dinit may still run successfully but any
 attempt to load a non-existing service, for example, will cause Dinit to crash.
+
+
+Special note for Clang (on systems using Libcxxrt C++ runtime)
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
+
+Clang on platforms that also use libcxxrt as the C++ runtime library has an issue that breaks
+exception handling and may prevent Dinit from working correctly when compiled with the "-fno-rtti"
+compiler option. The known platforms exhibiting this problem are FreeBSD and macOS.
+See "Recommended compiler options", above, for information about the "-fno-rtti" compiler option.
```

Comment:
> See "Recommended compiler options", above, for information about the "-fno-rtti" compiler option.

This comes too late, the reader is already confused. Move the reference inline is a parenthetical, s.t.l

    ... when compiled with the "-fno-rtti" compiler option (see "Recommended compiler options") ...